.travis.yml

```diff
@@ -4,5 +4,10 @@ jdk:
 before_install:
   - "echo $JAVA_OPTS"
   - "export JAVA_OPTS=-Xmx512m"
+
+script:
+  - mvn javadoc:jar
+  - mvn javadoc:test-aggregate
```

Comment:
I was concerned about that as well, but it looks like the default mvn build still runs. Here is the travis build from a similar commit to fcrepo-audit:
https://travis-ci.org/fcrepo4-labs/fcrepo-audit

@mikedurbin, have you tested your theory?